aries-site/src/examples/templates/content-layouts/ContainerDrivenLayout/ContainerDrivenLayout.js

```diff
@@ -59,72 +63,110 @@ export const ContainerDrivenLayout = ({ containerRef }) => {
         />
       </PageContent>
       <PageContent>
-        <ContentLayout containerRef={containerRef} />
+        <ContentLayout />
       </PageContent>
     </Page>
   );
 };
 
-ContainerDrivenLayout.propTypes = {
-  containerRef: PropTypes.object,
+const properties = {
+  categories: {
+    label: 'Categories',
+    options: [
+      { label: 'Machine Learning', value: 'Machine learning' },
+      { label: 'Big Data', value: 'Big Data' },
+      { label: 'Data Analytics', value: 'Data analytics' },
+      { label: 'Developer Tools', value: 'Developer tools' },
+      { label: 'Monitoring', value: 'Monitoring' },
+      { label: 'Database', value: 'Database' },
+      { label: 'Data Protection', value: 'Data Protection' },
+    ],
+  },
+  publisher: {
+    label: 'Publishers',
+  },
+  delivery: { label: 'Delivery Methods' },
+  pricing: { label: 'Pricing Models' },
 };
 
-const ContentLayout = ({ containerRef }) => {
-  // containerRef is for demonstration purposes on this site. Most
-  // implementations should likely remove.
-  const breakpoint = useContext(ResponsiveContext);
-  const [filteredApps, setFilteredApps] = useState(allApps);
-  const [filters, setFilters] = useState({});
-
-  // apply filters
-  useEffect(() => {
-    let results = [];
-    let numFilters = 0;
-    Object.values(filters).forEach(filter => (numFilters += filter.length));
+const filterButton = (label, expand, setExpand, property) => {
+  return (
+    <>
+      <Button
+        label={label}
+        icon={expand ? <Down /> : <Next />}
+        onClick={() => setExpand(!expand)}
+      />
+      <Collapsible open={expand}>
+        <Box pad={{ left: 'medium' }}>
```

Comment:
```suggestion
        <Box pad={{ left: 'medium', top: 'xsmall' }}>
```
Wondering if xsmall top padding might make this spacing look nicer:
<img width="709" alt="Screen Shot 2023-05-04 at 12 16 42 PM" src="https://user-images.githubusercontent.com/12522275/236306643-17b911e2-9806-41b7-b735-de390fc08228.png">

Reply:
I agree.